src/components/AgmExtension/AgmExtensionEvaluation.vue

```diff
@@ -84,21 +83,7 @@
           cols="12"
           sm="9"
         >
-          <template v-if="!data.isEligible">
-            <template v-if="data.alreadyExtended && !data.requestExpired">
-              The due date for this AGM cannot be set since extension has already been requested.
-            </template>
-            <template v-else-if="!data.alreadyExtended && data.requestExpired">
-              The due date for this AGM cannot be set since the request for extension has expired.
-            </template>
-            <template v-else-if="data.alreadyExtended && data.requestExpired">
-              The due date for this AGM cannot be set since extension has already been requested and the request
-              for extension has expired.
-            </template>
-          </template>
-          <template v-else>
-            {{ dueDateString }}
-          </template>
+          {{ dueDateString }}
```

Comment:
Wait... so we display a due date regardless of whether the business is eligible for an AGM extension?

@JazzarKarim @leodube-aot , will your component provide this value in all cases?

Reply:
Dove into this a bit. As it currently stands, we're not setting the `dueDate` anywhere. 

I'm pretty sure though that if the extension is ineligible, we won't be setting this date. This is because we need the `extensionDuration` to calculate the dueDate and that only can be gotten when the extension is eligible.

So, if eligible --> We'll show dueDate here.
If ineligible --> We'll show the date of extension expiry date here (According to UXPin).